.github/workflows/dotnet-core.yml

```diff
@@ -4,32 +4,42 @@ on:
   push:
     branches: [ 'master' ]
 
+env:
+  TRIBUTECH_FEED: https://pkgs.dev.azure.com/tributechsolutions/Tributech%20DataSpace%20Kit/_packaging/tributech-dsk/nuget/v3/index.json
```

Comment:
The Tributech DSK API clients are released to the public nuget.org feed (not AzureDevOps):
https://www.nuget.org/packages/Tributech.Dsk.Api.Clients/